src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -89,26 +88,19 @@
         buildPemFileDialog().choose().ifPresent(file -> setPemFile(Paths.get(file.toURI())))
     );
     pemFileClearButton.addActionListener(__ -> clearPemFile());
+
     startDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
-        startDatePicker.clear();
-      } else
-        onSelectStartDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
-    });
-    endDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
+
+      if (event.getNewDate() != null && isAfterEndDate(event.getNewDate()))
         endDatePicker.clear();
-      } else
-        onSelectEndDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
+
+      if (event.getNewDate() != null)
```

Comment:
You can remove the `if` block and directly set the date range limits with whatever comes in `event.getNewDate()` This way, when the user clears the field, the restriction is lifted.